src/api/app/policies/notification_policy.rb

```diff
@@ -1,6 +1,7 @@
 class NotificationPolicy < ApplicationPolicy
-  def update?
-    return true if record.subscriber_type == 'User' && record.subscriber_id == user.id
-    return true if record.subscriber_type == 'Group' && record.subscriber_id.include?(user.groups.ids)
+  class Scope < Scope
+    def resolve
+      scope.for_web.where(subscriber_type: 'User', subscriber_id: user.id).or(scope.where(subscriber_type: 'Group', subscriber_id: user.groups.ids))
```

Comment:
Could we use `NotificationsFinder` here? The `where(...)` is doing the same as `NotificationsFinder#for_subscribed_user`:

https://github.com/openSUSE/open-build-service/blob/ab37c65a7276d93a377a1cc0a5a33e87aba161d5/src/api/app/queries/notifications_finder.rb#L22-L25